dqc_us_rules/dqc_us_0005.py

```diff
@@ -7,33 +7,54 @@
 from arelle.XmlUtil import dateunionValue
 
 LEGALENTITYAXIS_DEFAULT = ''
-_dei_pattern = re.compile(r'^http://xbrl\.((sec\.gov)|(us))/dei/\d{4}-\d{2}-\d{2}')
+_dei_pattern = (
+    re.compile(r'^http://xbrl\.((sec\.gov)|(us))/dei/\d{4}-\d{2}-\d{2}'))
 _CODE_NAME = 'DQC.US.0005'
 _RULE_VERSION = '1.0'
 
 
 def _get_end_of_period(val):
     """
-    Gets the end of period date (dei:DocumentPeriodEndDate).  Returns the fact value or the context end date on the fact, whichever is later,
-    in a tuple with the fact itself and a string representation of the date. The tuple is in a dict keyed off of any LegalEntityAxis members.
-    The end of period string is adjusted for viewing to represent the day as expected.  In other words, dates that end in 24:00 will be put at 00:00 of the expected day.
+    Gets the end of period date (dei:DocumentPeriodEndDate).  Returns the fact
+    value or the context end date on the fact, whichever is later, in a tuple
+    with the fact itself and a string representation of the date. The tuple is
+    in a dict keyed off of any LegalEntityAxis members. The end of period
+    string is adjusted for viewing to represent the day as expected.  In other
+    words, dates that end in 24:00 will be put at 00:00 of the expected day.
 
-    @return A dictionary of tuples containing the fact, found date and a string representation of that date, keyed off of the LegalEntityAxis members in the format {lea_member: (fact, found_date, date_str)}
+    :param val: bal from which to gather end dates
+    :type val: val
+    :rtype: dictionary of tuples
```

Comment:
sg: ```:rtype: dict```
also it would be key: tuple, but you can leave it as dict then keep that info in the `:return:` field.